### ISSUE TITLE
[Accessibility issue] The h key alone opens the keyboard shortcut modal which can cause some users issues

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#74841]
**Steps to reproduce**:
When outside of any input, textarea or editable content area if you use the `h` key by itself it opens up the keyboard shortcuts modal.

**Actual Result**:
The keyboard shortcut modal appears.

**Expected Result**
As per the success criteria for [WCAG for 2.1.4 Character key shortcuts](https://www.w3.org/TR/WCAG21/#character-key-shortcuts) it should be possible for users to turn off this option, remap it or it is only invoked with a modifier key. An alternative option is to drop this shortcut as we also support `shift` + `?` to open up the keyboard shortcuts menu.

This issue can cause problems for users which use speech dictation instead of a keyboard, as well as users that might have motor and/or cognitive disabilities or users whose assistive technologies might have conflicts with single key character shortcuts.

UHG Audit ID: 2.14
